### PR TITLE
Browser check on sign-in

### PR DESCRIPTION
**Pull request by joshuarule**
Adds basic message to sign-in when browser isn't supported

### CODE REVIEW

File: ui/sign-in.reel/sign-in.js [hunk 1]
```diff
@@ -85,7 +89,7 @@ var SignIn = exports.SignIn = AuthorizationPanel.specialize({
         value: function () {
             this.addEventListener("action", this, false);
             this._keyComposer.addEventListener("keyPress", this, false);
-
+            this.isBrowserSupported = !!window.chrome && !!window.chrome.webstore;
```
Comment (thibaultzanini) [inline ui/sign-in.reel/sign-in.js]: There is an environment object in Montage that can help here.
https://github.com/montagejs/montage/blob/master/core/environment.js

File: ui/sign-in.reel/sign-in.html [hunk 2]
```diff
@@ -71,13 +71,25 @@
...
+        },
+
+        "browserMessage": {
+            "prototype": "montage/ui/text.reel",
+            "properties": {
+                "element": {"#": "browserMessage"},
+                "value": "We're sorry but at this time FreeNAS only supports the Chrome browser."
+            },
+            "bindings": {
+                "classList.has('hide')": {"<-": "@owner.isBrowserSupported"}
```
Comment (thibaultzanini) [inline ui/sign-in.reel/sign-in.html]: @pchaussalet are we still using the classname `hide` or `is-hidden`?

Comment (pchaussalet) [reply ui/sign-in.reel/sign-in.html]: It's more a question for @joshuarule... :)

Comment (joshuarule) [reply ui/sign-in.reel/sign-in.html]: I am using hide/show. `is-hidden` looks and sounds like a boolean in which I don't know the answer to. `hide` is a clear statement.

Let me know if you disagree.

Comment (pchaussalet) [reply ui/sign-in.reel/sign-in.html]: I have no strong opinion, but I think we mainly used is-\* or has-\* class names so far, which kept them consistent with properties names.
We may consider `hide` as an exception, as it is a class offered directly by montage, though...

Comment (joshuarule) [reply ui/sign-in.reel/sign-in.html]: You're right. I would rather keep everything consistent. I just need to wrap my head around the idea that every `is-*` is always a true statement in CSS land. I'm fine with using `is-hidden` `is-shown` `has-options`  etc... It's also a way a good way to visually see any css classname that is being toggled by js.

File: ui/sign-in.reel/sign-in.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 var AuthorizationPanel = require("montage-data/ui/authorization-panel.reel").AuthorizationPanel,
     Model = require("core/model/model").Model,
+    Environment = require("montage/core/environment").Environment,
```
Comment (thibaultzanini) [inline ui/sign-in.reel/sign-in.js]: @joshuarule The environment module exposes the following property `currentEnvironment`, so you don't need Line 27 to 40.

**Pull request merged**